README.md

```diff
@@ -26,3 +26,55 @@ Clean resources for testnet:
 ```bash
 make testnet-down
 ```
+
+## Appendix: Deploy and verify PasskeyAccountFactory on Sepolia testnet
+
+### Deploy PasskeyAccountFactory contract
+
+- Edit [.env](.env.example) and run the following command.
+
+```shell
+source .env
+
+forge create --rpc-url ${NODE_RPC_URL} --private-key ${DEPLOYER_PRIVATE_KEY} --use "0.8.23" "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory" --constructor-args ${ENTRYPOINT_ADDRESS}
+```
+
+- Output sample
+
+```shell
+[⠒] Compiling...
+[⠢] Compiling 4 files with 0.8.23
+[⠆] Solc 0.8.23 finished in 2.04s
+Compiler run successful!
+Deployer: 0x982A41a1F3bC1F8bdB71F11751F3a71691794AfA
+Deployed to: 0x4F0e62B7294D26b223a2cffc02BE5D072528c0De
+Transaction hash: 0xcf3c71f9b71fff6f7f45dbc3a512db667e10ab6eb272a700dfe1ab32aef0958c
+```
+
+### Verify PasskeyAccountFactory contract
+
+- Replace `<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>` with your deployed PasskeyAccountFactory address.
+
+```shell
+forge verify-contract --watch --chain "sepolia" --verifier "etherscan" --etherscan-api-key ${ETHERSCAN_API_KEY} --compiler-version "v0.8.23+commit.f704f362" --constructor-args $(cast abi-encode "constructor(address)" ${ENTRYPOINT_ADDRESS}) "<YOUR_PASSKEY_ACCOUNT_FACTORY_ADDRESS>" "src/account/PasskeyAccountFactory.sol:PasskeyAccountFactory"
```

Comment:
nit: Couldn't `--compiler-version` just assign `v0.8.23`?

Reply:
After testing, the `--compiler-version` parameter using either `"0.8.23"` or `"v0.8.23"` in the `forge verify-contract` can be recognized and successfully verify the contract.
Would it be better for us to consistency in using `"0.8.23"` matching the `--use` parameter in the preceding `forge create` command?

Reply:
Definitely. We can use 0.8.23 for both places 👍